src/content/docs/apis/nerdgraph/examples/nerdgraph-synthetics-tutorial.mdx

```diff
@@ -0,0 +1,745 @@
+
+Below are some sample scripts to automate the creation of your synthetic monitors. Unlike with REST APIs, NerdGraph lets you programatically create [broken links](#create-broken-links), [step](#create-step), and [certificate check](create-certificate-check) monitors. 
+
+<CollapserGroup>
+  <Collapser
+    className="freq-link"
+    id="create-ping"
+    title="Create a ping monitor"
+  >
+  Ping monitors check that your application is running: 
+
+```
+  mutation {
+    syntheticsCreateSimpleMonitor (
+      accountId: <var>YOUR_ACCOUNT_ID</var> 
+      monitor: {
+        locations: {
+          public: [<var>LOCATION_1</var>, <var>LOCATION_2</var>]
+          }, 
+        name: <var>YOUR_MONITOR_NAME</var>, 
+        period: <var>PERIOD</var>, 
+        status: <var>STATUS</var>, 
+        uri: <var>MONITORED_URI</var>, 
+        advancedOptions: {
+          customHeaders: {
+            name: <var>HEADER_NAME</var>, 
+            value: <var>HEADER_VALUE</var>
+          }, 
+          redirectIsFailure: <var>REDIRECT_IS_FAILURE</var>, 
+          responseValidationText: <var>VALIDATION_TEXT</var>, 
+          shouldBypassHeadRequest: <var>BYPASS_HEAD_REQUEST</var>, 
+          useTlsValidation: <var>TLS_VALIDATION</var>
+        }
+      }
+  ) {
+      errors {
+        description
+        type
+      }
+    }
+  }
+```
+</Collapser>
+<Collapser
+    className="freq-link"
+    id="create-simple-browser"
+    title="Create a simple browser monitor"
+  >
+Make some simple browser monitors for instances of Google Chrome that mimic customer visits:
+
+```
+  mutation {
+    syntheticsCreateSimpleBrowserMonitor (
+      accountId: <var>YOUR_ACCOUNT_ID</var>, 
+      monitor: {
+        browserType: <var>BROWSER_TYPE</var>, 
+        browserVersion: <var>BROWSER_VERSION</var>, 
+        locations: {
+          public: [<var>LOCATION_1</var>, <var>LOCATION_2</var>]
+          }, 
+        name: <var>YOUR_MONITOR_NAME</var>, 
+        period: <var>PERIOD</var>, 
+        scriptLanguage: <var>SCRIPT_LANGUAGE</var>, 
+        status: <var>STATUS</var>, 
+        uri: <var>MONITORED_URI</var>, 
+        advancedOptions: {
+          customHeaders: {
+            name: <var>HEADER_NAME</var>, 
+            value: <var>HEADER_VALUE</var>
+          }, 
+          enableScreenshotOnFailureAndScript: <var>ENABLE_SCREENSHOT</var>, 
+          responseValidationText: <var>VALIDATION_TEXT</var>, 
+          useTlsValidation: <var>TLS_VALIDATION</var>
+        }
+      ) {
+      errors {
+        description
+        type
+      }
+    }
+  }
+```
+</Collapser>
+
+<Collapser
+    className="freq-link"
+    id="create-scripted-browser"
+    title="Create a scripted browser"
+  >
+
+  [Scripted browser monitors](/docs/synthetics/synthetic-monitoring/scripting-monitors/introduction-scripted-browser-monitors) are more complex interactions with your pages and applications. They take specific actions to navigate your website and check for specific resources:
```

Comment:
in keeping with previous comments, I'd add the kind of 'Here's an example of...' type wording